tools/rna_tools/targetfinder/TargetFinder.xml

```diff
@@ -1,73 +1,236 @@
-<tool id="targetfinder" name="Plant small RNA target prediction tool" version="1.7.0">
```

Comment:
Cris please note that the old version was 1.7.0 the new one will be 1.7 ... this is probably lower? Not sure. Maybe we need to hardcode the version here to 1.7.0+galaxy1 or something like that to make sure its really bigger than the previous one.